### ISSUE TITLE
attention mask and the pad token id were not set warning

### ISSUE THREAD

**Issue by adityarajsahu** [suno-ai/bark#539]
I am getting this warning while trying to generate audio from text prompts

`
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's attention_mask to obtain reliable results. Setting pad_token_id to eos_token_id:10000 for open-end generation.
`

And the generated audio is very noisy and not up to the mark. How do I resolve this issue?

**Comment by nischalj10** [comment]
Prior discussion #402
